cmd/format-fs.go

```diff
@@ -86,6 +88,16 @@ func formatMetaGetFormatBackendFS(r io.ReadSeeker) (string, error) {
 	return "", fmt.Errorf(`format.Version expected: %s, got: %s`, formatMetaVersionV1, format.Version)
 }
 
+// Returns the format JSON.
+// We do not use this function in XL to get the format as the file is not fcntl-locked on XL.
+func formatGetFormatJSON(format *formatFSV2, r io.ReadSeeker) error {
+	if format == nil {
+		return fmt.Errorf("format is nil")
+	}
+	err := jsonLoad(r, format)
```

Comment:
It would be little easier if you return here `return jsonLoad(r, format)`

Reply:
rewriting this logic. So, this function will be removed.